reward/stakingManager_test.go

```diff
@@ -0,0 +1,43 @@
+package reward
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+// checking calculate blockNumber of stakingInfo and return the stakingInfo with the blockNumber correct when stakingInfo is stored in cache
+func TestStakingManager_getStakingInfoFromStakingCache(t *testing.T) {
+	testData := []uint64{
+		0, 86400, 172800, 259200,
```

Comment:
Can we add a comment for describing why those numbers are meaningful?

Reply:
86400 = duration of epoch? If so, how about declaring a variable `epoch` and multiply 0, 1, 2, 3 with that?

Reply:
reflected